jsdrupal/drupal-admin-ui
# 506: NumberTextField: Mishandled unsigned property

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/506

## Screenshot / UI changes

Storybook entry for NumberTextField throws exceptions

## Testing instructions

yarn run storybook 

See that NumberTextField now displays correctly.

Please note that as per 

https://github.com/jsdrupal/drupal-admin-ui/issues/506#issuecomment-431794761

I cannot justify why unsigned exists in the first place.
